setup.py

```diff
@@ -36,6 +36,8 @@ def get_all_extras() -> Dict:
         "pyyaml>=4.2b1,<6.0",
         "jsonschema>=3.0.0,<4.0.0",
         "packaging>=20.3,<22.0",
+        "pytest>=7.0.0,<7.2.0",
+        "coverage>=6.4.4,<7.0.0",
         "semver>=2.9.1,<3.0.0",
     ]
 
```

Comment:
should not we add more dependencies in the `test_tools` extra (e.g. `docker`), due to the recent changes to `aea/test_tools`?

Reply:
So `docker` is used in a way to not having to make it a dependency in the framework (lazy imports etc). This way, the user can install it when they actually use test tools!